### PR TITLE
Add id_type config

### PR DESCRIPTION
**Pull request by jdiddydave**
Adds id_type config to Verbs. Adds match expressions to migrations, defaulting to snowflakes.
It also modifies the key types on VerbsSnapshot.php

### CODE REVIEW

File: database/migrations/create_verb_snapshots_table.php [hunk 1]
```diff
@@ -9,7 +9,11 @@
     public function up()
     {
         Schema::create('verb_snapshots', function (Blueprint $table) {
-            $table->snowflakeId();
+            match (config('verbs.id_type', 'snowflake')) {
+                'snowflake' => $table->snowflakeId(),
```
Comment (inxilpro) [inline database/migrations/create_verb_snapshots_table.php:13]: Should this be the default branch? If not, we may want to throw an exception if it's not set to something we expect.

```suggestion
                default => $table->snowflakeId(),
```

File: src/Models/VerbSnapshot.php [hunk 2]
```diff
@@ -40,4 +40,22 @@ public function scopeWhereDataContains($query, array $data)
...
+    //
+
+    public function getKeyType()
+    {
+        return match (config('verbs.id_type', 'snowflake')) {
+            'snowflake' => 'int',
+            'ulid' => 'string',
+            'uuid' => 'string',
+        };
+    }
```
Comment (inxilpro) [inline src/Models/VerbSnapshot.php:55]: I think this should happen on the other two models, too?

Comment (DanielCoulbourne) [reply src/Models/VerbSnapshot.php:55]: But why though, sir?

Comment (inxilpro) [reply src/Models/VerbSnapshot.php]: lol I guess you're right — sorry! I see that y'all are still working on this and I was just going to resolve my comments and merge it. Probably introduced a merge conflict :(

**Pull request merged**